src/UpgradeableProxied.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
```

Comment:
I think we can relax the version requirement to `^0.8.0`.

Reply:
Updated